src/test/Components/CreateImageWizardV2/steps/FirstBoot/Firstboot.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+import {
+  CREATE_BLUEPRINT,
+  FIRST_BOOT_SERVICE_DATA,
+} from '../../../../../constants';
+import { File as ImageBuilderFile } from '../../../../../store/imageBuilderApi';
+import { clickNext } from '../../../../testUtils';
+import {
+  blueprintRequest,
+  clickRegisterLater,
+  enterBlueprintName,
+  interceptBlueprintRequest,
+  render,
+} from '../../wizardTestUtils';
+
+import '@testing-library/jest-dom';
+
+jest.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  useChrome: () => ({
+    auth: {
+      getUser: () => {
+        return {
+          identity: {
+            internal: {
+              org_id: 5,
+            },
+          },
+        };
+      },
+    },
+    isBeta: () => true,
+    isProd: () => true,
+    getEnvironment: () => 'prod',
+  }),
+}));
+
+const goToFirstBootStep = async () => {
+  const bareMetalCheckBox = await screen.findByRole('checkbox', {
+    name: /bare metal installer checkbox/i,
+  });
+  await userEvent.click(bareMetalCheckBox);
+  await clickNext(); // Registration
+  await clickRegisterLater();
+  await clickNext(); // OpenSCAP
+  await clickNext(); // File System
+  await clickNext(); // Custom repositories
+  await clickNext(); // Additional packages
+  await clickNext(); // First Boot
+};
+
+const goToReviewStep = async () => {
+  await clickNext(); // Details
+  await enterBlueprintName();
+  await clickNext(); // Review
+};
+
+const SCRIPT = `#!/bin/bash
+systemctl enable cockpit.socket`;
+
+const getFirstbootData = (): ImageBuilderFile[] => [
```

Comment:
Would it be simpler for now to have this just be an object and not a function that returns an object?